AnimalsTest/Lion.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace AnimalsTest{
+    
+    public class Lion : IBigCats
+    {
+
+        private string? _name;
+        public string? Name
+        {
+            get
+            {
+                return _name;
+            }
+            set
+            {
+                if (!IsValidName(value))
+                {
+
+                    _name = "Baoka";
+                }
+                else
+                {
+                    _name = value;
+                }
+
+            }
+        } 
+
+        public bool IsValidName(string? name)
+        {
+            Regex expression = new Regex(@"^[a-zA-Z]+$");
+            if (!string.IsNullOrEmpty(name))
+            {
+                return expression.IsMatch(name);
+            }
+            return false;
+
+        }
```

Comment:
This is exactly same as in `Jaguar`. Consider doing base class for it (we can talk on Fri about this).